icons/square-slash.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "../icon.schema.json",
+  "tags": [
+    "git",
+    "diff",
+    "ignored"
```

Comment:
While this is a nice icon to have, I'm not sure how universal these use cases are. 🤔

Reply:
What do you mean exactly? Well it is `diff-ignored` in both octicons and codicons… Are you proposing to remove these `tags`? Shouldn’t we be aiming to make search as useful as possible?

Reply:
Search should be useful, I'm just not entirely sure if this shape truly has  this universal meaning.

It's especially funny for GitHub to call this "diff ignored", when in actuality they're using it for slash commands. 😄 

![D659C218-EC4F-4C8D-A19D-3B249BC92A68](https://user-images.githubusercontent.com/17746067/234471586-439b77bf-342f-4fd9-884c-a5b58454bf56.jpeg)

Reply:
Yeah I noticed that before. It’s used for both (like a good, useful icon should be)… pretty sure Iv’e seen it in GitHub Desktop and Atom also.

> not entirely sure if this shape truly has this universal meaning.

Isn’t that the whole point of `tags` though—to include other possible (relevant) meanings? By which logic:
Obviously `slash` is part of the file name already, but maybe we should also add `commands` as a tag then…?